packages/mocks/src/definitions/basic/secrets.mock.ts

```diff
@@ -0,0 +1,45 @@
+import { ISecret } from '@kubev2v/types';
+
+export let MOCK_SECRET_INSECURE: ISecret;
+export let MOCK_SECRET_SECURE: ISecret;
+
+if (process.env.NODE_ENV === 'test' || process.env.DATA_SOURCE === 'mock') {
```

Comment:
This kind of code should be removed...well once the queries that reference them from `useMockableQuery()` are moved to a normal `useQuery()`.  MSW handlers will need to wrap some thing up in container objects since `useMockableQuery()` does that already.

The provider's handlers I have in #392 already do this.  Hopefully it is an easy standard to follow.

Reply:
I've missed this one. Thanks!